fitlins/cli/run.py

```diff
@@ -135,9 +135,7 @@ def create_workflow(opts):
     deriv_dir = op.join(output_dir, 'fitlins')
     os.makedirs(deriv_dir, exist_ok=True)
```

Comment:
You can omit this line, since the dir is also made in `write_derivative_description`

Reply:
Does it make more sense for `write_derivative_description` to assume that `deriv_dir` exists?

Reply:
Probably, although that's what's nice was `exist_ok`, is you can use it either way.

Reply:
If you think in the future there may be other files you might want to instantiate in the folder, you could call the function something like `create_derivative_template` or something

Reply:
Okay. For now I'm just going to let `create_workflow` do the `makedirs`. Might change again in the future.